src/java.desktop/share/classes/java/awt/LayoutManager2.java

```diff
@@ -25,14 +25,14 @@
 package java.awt;
 
 /**
- * Defines an interface for classes that know how to layout Containers
+ * Defines an interface for classes that know how to layout {@code Container}s
  * based on a layout constraints object.
  *
- * This interface extends the LayoutManager interface to deal with layouts
+ * This interface extends the {@code LayoutManager} interface to deal with layouts
  * explicitly in terms of constraint objects that specify how and where
  * components should be added to the layout.
  * <p>
- * This minimal extension to LayoutManager is intended for tool
+ * This minimal extension to {@code LayoutManager} is intended for tool
  * providers who wish to the creation of constraint-based layouts.
```

Comment:
Where did your first change disappear now?

Reply:
Sorry while updating I have missed it.